src/libtiled/tilelayer.h

```diff
@@ -266,7 +267,7 @@ class TILEDSHARED_EXPORT TileLayer : public Layer
      * are rotated within the layer, and the tiles themselves are rotated. The
      * dimensions of the tile layer are swapped.
```

Comment:
Please still adjust this comment, and make a note of the `map` parameter and that the stagger axis may be changed.